src/ZoneServer/ZoneServer.cs

```diff
@@ -101,7 +101,11 @@ public override void Run(string[] args)
 			var title = string.Format("Zone ({0}, {1})", groupId, serverId);
 
 			ConsoleUtil.WriteHeader(ConsoleHeader.ProjectName, title, ConsoleColor.DarkGreen, ConsoleHeader.Logo, ConsoleHeader.Credits);
-			ConsoleUtil.LoadingTitle();
+			
+			// Skip the following command on incompatible systems
+			if (Environment.OSVersion.Platform == PlatformID.Win32NT)
```

Comment:
We since fixed the problem with `LoadingTitle` under Linux, and this should now work without any exceptions made in the code.